### PR TITLE
Add Are We Synapse Yet to GHA

### PR DESCRIPTION
**Pull request by S7evinK**
Even if already run from Sytest, I think this makes it easier to get a fast overview (without having to scroll down).
Are We Synapse Yet runs in verbose mode and groups the tests.

### CODE REVIEW

File: .github/workflows/dendrite.yml [hunk 1]
```diff
@@ -233,7 +233,14 @@ jobs:
       - name: Summarise results.tap
         if: ${{ always() }}
         run: /sytest/scripts/tap_to_gha.pl /logs/results.tap
-
+      - name: Sytest List Maintenance
+        if: ${{ always() }}
```
Comment (kegsay) [inline .github/workflows/dendrite.yml:237]: What's the deal with this? I've not seen this before.

**Review by kegsay — APPROVED** [review]
Looks great!

File: .github/workflows/dendrite.yml (hunk 1 above)
Comment (S7evinK) [reply .github/workflows/dendrite.yml:237]: The list maintenance has already been in Buildkite, but somehow got lost over time.
Not sure if the `if: ${{ always() }}` should be left in or if we want this to fail the build.

Comment (neilalexander) [reply .github/workflows/dendrite.yml:237]: FWIW I don't think things that are purely informational should fail the pipeline.

**Pull request merged**